### ISSUE TITLE
Spell checker for note editor

### ISSUE THREAD

**Issue by jlbrewer** [turtl/tracker#215]
Pretty self-explanatory I hope. This is a thing that would go some way toward making Turtl feature-competitive with stuff like Evernote. The main downside is that to do this without having to reinvent the wheel you would probably have to turn to 3rd party libraries and I saw the note about why you are very careful about that. I know the names of the major players (aspell, ispell, hunspell); I don't know anything about any security concerns inherent with them. I would hope however, that they are used in enough big open-source projects (the major browsers, word processors, etc) that at least one of them will be sufficiently well-vetted.

**Comment by orthecreedence** [comment]
Yes, I think a fully-bundled open-source option could work here.
